### PR TITLE
DCGEO-88 (DRAFT): FISS/MCS claim schema updates

### PR DESCRIPTION
**Pull request by brianburton**
**NOTE: This is a DRAFT and WIP and not ready for review.**

### Change Details

Adds Flyway schema migration script and associated JPA entities for the following:

- 4 new practice location related fields in FissClaims
- new FissDiagnosisCodes table for FISS claim diagnosis codes
- new McsClaims table for MCS claim data
- new McsDetails table for MCS claim details
- new McsDiagnosisCodes table for MCS diagnosis codes

Also adds some HSQLDB compatibility changes to existing entities and some an IT test class to verify basic database functionality.

### Acceptance Validation

Is the SQL migration correct and are the JPA entities compatible with the tables.

### Feedback Requested

nothing specific

### External References

- [DCGEO-88](https://jira.cms.gov/browse/DCGEO-88)
- [RDA API tech spec for schema](https://confluenceent.cms.gov/display/MPSM/2021-04-22+Tech+Spec%3A+Specific+Schema+for+MVP+v0.2)

### Security Implications

Adds new tables to the database.

- [ ] new software dependencies

no

- [ ] altered security controls

no

- [ ] new data stored or transmitted

new FISS fields and MCS claim records

- [ ] security checklist is completed for this change

no

- [ ] requires more information or team discussion to evaluate security implications

no

### CODE REVIEW

File: apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjMcsClaim.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.FieldNameConstants;
+
+@Entity
+@Getter
+@Setter
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@FieldNameConstants
+@Table(name = "`McsClaims`", schema = "`pre_adj`")
```
Comment (ericbuckley) [inline apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjMcsClaim.java:23]: I think we're missing `idrTotBilledAmt`

File: apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjMcsClaim.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  @Column(name = "`idrBeneSex`", length = 1)
+  private String idrBeneSex;
+
+  @Column(name = "`idrStatusCode`", length = 1)
+  private String idrStatusCode;
+
+  @Column(name = "`idrStatusDate`")
+  private LocalDate idrStatusDate;
+
+  @Column(name = "`idrBillProvNp_1_6`", length = 10)
```
Comment (ericbuckley) [inline apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjMcsClaim.java:60]: is this supposed to be `IDR-BILL-PROV-NPI`?

File: apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjMcsClaim.java (hunk 1 above)
Comment (ericbuckley) [reply apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjMcsClaim.java:23]: and `idrClaimMbi`

Comment (brianburton) [reply apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjMcsClaim.java:24]: Good catch.  My script didn't convert the decimal columns.  I have pushed a fix with the extra 4 fields.

File: apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjMcsClaim.java (hunk 2 above)
Comment (brianburton) [reply apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjMcsClaim.java]: Yes.  Fixed.

**Comment by ericbuckley** [comment]
this PR is no longer needed, all necessary changes were merged in with #677

**Pull request closed without merging**